app/app_test.go

```diff
@@ -627,14 +483,11 @@ func TestProvisionDevice(t *testing.T) {
 		t.Run(tc.Name, func(t *testing.T) {
 			t.Parallel()
 			ctx := context.Background()
+
 			ds := tc.Store(t, &tc)
-			hub := tc.Hub(t, &tc)
-			wf := tc.Wf(t, &tc)
 			defer ds.AssertExpectations(t)
-			defer hub.AssertExpectations(t)
-			defer wf.AssertExpectations(t)
 
-			app := New(ds, hub, wf, nil)
+			app := New(ds, nil, nil)
```

Comment:
why don't we need .With*(hub) here?

Reply:
No, because I moved the hub-specific tests to app_iot_hub_test.go